views/index.jade

```diff
@@ -48,12 +48,29 @@ html(lang='en')
               | If you are a Mozilla employee, sign in with Okta to maximize your
               | permissions.  If you also have a Mozillians account, make sure your
               | Okta email is included in your Mozillians profile.
-        div.row
+        div.row.login-option
+          div.col-md-4
+            form#ldap-form(action='/ldap/login', method='post')
+              label.sr-only(for="username") LDAP Email
+              input.form-control#username(name="username", type="email",
+                                          placeholder="LDAP Email", required)
+              label.sr-only(for="password") Password
+              input.form-control#password(name="password", type="password",
+                                          placeholder="Password", required)
+
+              button.btn.btn-primary(type='submit')
+                i.glyphicon.glyphicon-circle-arrow-right
+                | &nbsp;Sign-in with LDAP
+          div.col-md-8
+            p
+              | If you are not a Mozilla employee, but have an LDAP account granting
+              | you commit access, sign in with your LDAP username and password.
+        div.row.login-option
```

Comment:
Aren't you also missing an end-point for this... I think the PR does more than just flash...

Anyways, feel free to add ldap login in the same round :)